### PR TITLE
Vietnam

### PR DESCRIPTION
**Pull request by soma1020**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2485)

### CODE REVIEW

File: vietnamese/tien-giang/tien-giang.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (casey-collab) [inline vietnamese/tien-giang/tien-giang.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by soma1020** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fireworks:

**Pull request merged**